src/components/forms/api.ts

```diff
@@ -150,6 +152,27 @@ export const api = {
       return fetch(API_DEVCON_GRANTS, devconGrantsRequestOptions);
     }
   },
+  mergeDataChallenge: {
+    submit: (data: MergeDataChallengeFormData) => {
+      const mergeDataChallengeRequestOptions: RequestInit = {
+        ...methodOptions,
+        body: JSON.stringify({
+          ...data,
+          company: data.company === 'N/A' ? `${data.firstName} ${data.lastName}` : data.company,
+          blogPostURL: getWebsite(data.blogPostURL),
+          applyingAs: data.applyingAs.value,
+          country: data.country.value,
+          timezone: data.timezone.value,
+          projectCategory: data.projectCategory.value,
+          wouldYouShareYourResearch: data.wouldYouShareYourResearch.value,
+          repeatApplicant: data.repeatApplicant === 'Yes',
+          canTheEFReachOut: data.canTheEFReachOut === 'Yes'
```

Comment:
@corwintines I think we need to grab the `.value` from the referral source here
```suggestion
          canTheEFReachOut: data.canTheEFReachOut === 'Yes',
          howDidYouHearAboutESP: data.howDidYouHearAboutESP.value
```

Reply:
This should then be a string, allowing the `.trim()` method